From c8347b12035ffd080ae6252da2a2258132312891 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:09 -0400
Subject: [redhat] kabi: Adding symbol net_namespace_list (net/core/net_namespace.c)

Message-id: <3cfd94072ed3d86683cd0f4ecafa3ca8471ae5ee.1410510773.git.poros@redhat.com>
Patchwork-id: 96073
O-Subject: [PATCH RHEL7.1 129/233] kabi: Adding symbol net_namespace_list (net/core/net_namespace.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Our VEM takes over the ports in different name space.
    We use 'for_each_net' to find which name-space the port belongs to.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/net_namespace_list
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/net_namespace_list
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/net_namespace_list

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/net_namespace_list b/redhat/kabi/kabi-module/kabi_ppc64/net_namespace_list
new file mode 100644
index 0000000..cad85b3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/net_namespace_list
@@ -0,0 +1,2 @@
+#1-
+0x6d2fc5a6 net_namespace_list vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/net_namespace_list b/redhat/kabi/kabi-module/kabi_s390x/net_namespace_list
new file mode 100644
index 0000000..cad85b3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/net_namespace_list
@@ -0,0 +1,2 @@
+#1-
+0x6d2fc5a6 net_namespace_list vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/net_namespace_list b/redhat/kabi/kabi-module/kabi_x86_64/net_namespace_list
new file mode 100644
index 0000000..cad85b3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/net_namespace_list
@@ -0,0 +1,2 @@
+#1-
+0x6d2fc5a6 net_namespace_list vmlinux EXPORT_SYMBOL_GPL
-- 
1.7.1